.github/workflows/continuous-build.yml

```diff
@@ -35,7 +35,7 @@ jobs:
           echo echo waiting for dockers to startup
           while [ $i -le 2 ]
           do
-            if (docker logs $(docker ps -f name="yoma-ga-agent" | awk 'FNR == 2 {print $1}') | grep -q "::::::::::::::::::::::::::::::::::::::::::::::
+            if (docker logs $(docker ps -f name="governance-acapy-agent" | awk 'FNR == 2 {print $1}') | grep -q "::::::::::::::::::::::::::::::::::::::::::::::
             :: Alice                                    ::
```

Comment:
I see the label `Alice` has been replaced in other places. Shouldn't this break now as it's excplitly looking for `Alice`

Reply:
hmm fair point. Not sure why that still works then actually